### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheSteelBarr**
Added a more detailed description in [README](readme.md) plus Copyright details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [![Stories in Needs Review](https://badge.waffle.io/Micathon3/Zeus-Web.svg?label=needs%20review&title=Needs%20Review)](http://waffle.io/Micathon3/Zeus-Web)
 
 # Zeus-A3
-Zeus A3 is a computer OS (of sorts).
+Zeus A3 is a computer OS (of sorts). It is one of many created by Micathon in the Zeus line. From 2016, Micathon and all of its products are under THE DIGITAL MILLENNIUM COPYRIGHT ACT OF 1998. You can view more here: https://www.copyright.gov/legislation/dmca.pdf
```
Comment (StellarDoor5319) [inline README.md]: Should also include a link to the LICENCE file. [See here](LICENCE).

Comment (StellarDoor5319) [inline README.md]: The link should be written as a Markdown link, like so:

```Markdown
You can view more [here](https://www.copyright.gov/legislation/dmca.pdf).
```

**Comment by StellarDoor5319** [comment]
@TheSteelBarr Remember to choose a descriptive branch name, rather than just `TheSteelBarr-update-1`.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 [![Stories in Needs Review](https://badge.waffle.io/Micathon3/Zeus-Web.svg?label=needs%20review&title=Needs%20Review)](http://waffle.io/Micathon3/Zeus-Web)
 
 # Zeus-A3
-Zeus A3 is a computer OS (of sorts).
+Zeus A3 is a computer OS (of sorts). It is one of many created by Micathon in the Zeus line. From 2016, Micathon and all of its products are under THE DIGITAL MILLENNIUM COPYRIGHT ACT OF 1998. You can learn more [here](https://www.copyright.gov/legislation/dmca.pdf), view the Copyright Act [here](https://creativecommons.org/licenses/by-nc-nd/4.0/) view the MIT licence [here](https://github.com/Micathon3/Zeus-A3/blob/master/LICENSE)
```
Comment (StellarDoor5319) [inline README.md]: Looking good now.

**Comment by MicathonTracker** [comment]
@StellarDoor5319, I gave you **0.25333333333333335 XP** for your review. You now have 0.25333333333333335 XP in total. Keep it up!

**Pull request merged**